Doc/using/configure.rst

```diff
@@ -619,10 +713,22 @@ Compiler flags
 
    .. versionadded:: 3.8
 
+.. envvar:: PURIFY
+
+   Purify command. Purify is a memory debugger program.
```

Comment:
maybe add a link to the official purify website (looks like a closed source tool, right?)

Reply:
I found https://en.wikipedia.org/wiki/PurifyPlus but I'm not sure if it's this software. In case of doubt, I prefer to omit it.

Yeah, it's closed source. GCC and Clang have now great open source implementations (ASAN, UBSAN, etc.) supported by Python, see the Debug Options in this document.